hw/ip_templates/clkmgr/dv/tb.sv.tpl

```diff
@@ -157,13 +158,14 @@ module tb;
     .lc_clk_byp_req_i(clkmgr_if.lc_clk_byp_req),
     .lc_clk_byp_ack_o(clkmgr_if.lc_clk_byp_ack),
     .div_step_down_req_i(clkmgr_if.div_step_down_req),
-
+    .calib_rdy_i(clkmgr_if.calib_rdy),
+  
+  % endif
     .cg_en_o(),
 
     .jitter_en_o(clkmgr_if.jitter_en_o),
     .clocks_o   (clkmgr_if.clocks_o),
 
-    .calib_rdy_i(clkmgr_if.calib_rdy),
```

Comment:
You didn't remove calib_rdy_i from clkmgr.sv, so you should keep it in the interface.

It is a separate question whether you still need calib_rdy. If you expect the clocks in SCS to be always calibrated then it is not needed. However, this calib_rdy is also tied to the clock measurement functionality: if the clocks loose calibration for some reason then calib_rdy == 0 will disable measurements and make the measure control CSRs to become writeable.